### ISSUE TITLE
Tabs component does not communicate role or state

### ISSUE THREAD

**Issue by dougmacknz** [cultureamp/kaizen-design-system#1934]
Our Tabs component could do a lot better at rendering more semantic markup. We're currently rendering `<a>` tags regardless of whether `hrefs` are actually provided, and there's some additional best practices that we can add (see links below).

## Audit info

Type:WCAG 2
Principle: Robust
Guideline:4.1 Compatible
Success Criteria: 4.1.2 Name, Role, Value (A)
Level: A
Platform: Web
Severity:Critical

### What's the issue?
The tabs component is marked up using a <a> element. It does not communicate its role (tab) or state (selected/unselected).

### Why it matters
If a tab does not have the tab role assigned to it then a screen reader user will not know how to interact with it.
A screen reader user will not know the current state of the tab

### Our recommendation

See the [Intopia ARIA Tabs codepen](https://codepen.io/intopia/pen/eYZmvrN) for an example.

See the [ARIA Practices Tabs example](https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-2/tabs.html) for information on the ARIA tab component including markup and keyboard functionality.

**Comment by ActuallyACat** [comment]
How important is the order of the content? is it enough to markup the tabs with `aria-control=blah` with the related content as `id=blah`.
